### DISCUSSION TITLE
Econ-ARK readme updates

### DISCUSSION THREAD

**Discussion by DominicWC** [econ-ark/HARK#1450] [General]
On the econ-ark GitHub page when this post was made (6/06/2024), the links to the donation page, the TSC/governance of econ-ark, and the HARK manual are all broken. In my pull request #1436, I've removed them from the docs and the readme. What should they be replaced with?

I'm currently directing users to the glitter instead of the TSC. Is that what we want to encourage?

The suggested citation is from 2018, should it be updated?

**Comment by mnwhite** [comment]
Dominic and I discussed this in a Zoom session. He's fixing it.
